moveit_ros/hybrid_planning/test/config/common_hybrid_planning_params.yaml

```diff
@@ -1,6 +1,6 @@
 # Parameters that are shared across several component nodes
 
 # A namespace can be added if multiple hybrid planners are launched
-global_planning_action_name: "/test/hybrid_planning/global_planning_action"
-local_planning_action_name: "/test/hybrid_planning/local_planning_action"
-hybrid_planning_action_name: "/test/hybrid_planning/run_hybrid_planning"
+global_planning_action_name: "/hybrid_planning/global_planning_action"
```

Comment:
Is there a strong need to make these configurable? Technically, it would be cleaner to remove the action names here and use constants. It's a debated ROS best practice to not configure topic or action names since they are part of the runtime interface. Instead, we could rely on node namespaces and topic remapping.